buildlib/cbuild

```diff
@@ -171,6 +171,65 @@ class centos7_epel(centos7):
         res.lines.append("RUN ln -s /usr/bin/cmake3 /usr/local/bin/cmake && ln -sf /usr/bin/python3.4 /usr/bin/python3");
         return res;
 
+class amazonlinux1(YumEnvironment):
+    docker_parent = "amazonlinux:1";
```

Comment:
Do you really need all three? How different is this actually from Centos?

Reply:
I'm not part of the AmazonLinux distro team so I really can't say how different they are from CentOS.
Why not have all three? Is there a goal to reduce the number of images?

Reply:
It takes a long time to build all these images for testing purposes, if they are pointless I prefer not to have them all. ie checking the same EPEL modified packages against amazon and c7 is probably redundant.

I'd suggest the latest image only..

Reply:
OK, I'll remove amazonlinux2 and keep amazonlinux1 and amazonlix2_epel (which will be renamed to amazonlinux2).

Reply:
Ended up removing amazonlinux2_epel.